indico/web/client/js/legacy/libs/timetable/Draw.js

```diff
@@ -147,6 +147,9 @@ type(
       if (this.eventData.slotTitle && this.eventData.slotTitle !== '') {
         title += ': ' + this.eventData.slotTitle;
       }
+      if (this.eventData.code != null) {
```

Comment:
```suggestion
      if (this.eventData.code) {
```

It's an empty string, not null